src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +794,86 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        deployment_configs = self.list_deployment_configs()
+
+        data = {"Config Name": [], "Instance Type": [], "Selected": [], "Accelerated": []}
+
+        for i in range(0, len(deployment_configs)):
```

Comment:
nit: this is no diff than `for i in range(len(deployment_configs))` i think. default starting point is 0